trunk/source/Library-UltimateModel/src/de/uni_freiburg/informatik/ultimate/core/model/preferences/UltimatePreferenceItem.java

```diff
@@ -203,6 +206,38 @@ public String getInvalidValueErrorMessage(final Double value) {
 				return "Valid range is " + mMin + " <= value <= " + mMax;
 			}
 		}
+		
+		public class StringValidator implements IUltimatePreferenceItemValidator<String> {
+
+			private final String mPattern;
+
+			public StringValidator(String pattern) {
+				mPattern = pattern;
+			}
+
+			@Override
+			public boolean isValid(final String string) {
+				String[]exprPairs =   string.split(",");

```

Comment:
Please remove white space repetition:

```suggestion
				String[]exprPairs = string.split(",");

```